#164198494 migration & model for article, helper functions for responses.

danprocoder opened the pull request:
#### What does this PR do?
Creates helper functions for responses  and create migration and model for articles.

#### Description of Task to be completed?
- Created migrations and models
- Created Controller class with helper functions for responses to prevent repetition

#### What are the relevant pivotal tracker stories?
[#164198494](https://pivotaltracker.com/story/show/164198494)

danprocoder committed:
chore(createArticles): feature to create articles

- Created migrations and models
- Created Controller class with helper functions for responses to prevent repetition

[Delivers #164198494]

danprocoder committed:
chore(createArticles): feature to create articles

- Created migrations and models
- Created Controller class with helper functions for responses to prevent repetition

[Delivers #164198494]

danprocoder committed:
chore(controllerBaseClass): Fixed a merge conflict

[Finishes #164198494]

andela-gike reviewed (changes requested):
Your file structure is not very descriptive, `controllers/controller.js`; what form of controller is it, please kindly confirm and make the changes where appropriate

danprocoder committed:
Merge branch 'develop' of https://github.com/andela/artemis-ah-backend into ft-create-article-164198494

danprocoder committed:
chore(*): Moved server files to ./server folder

[Finishes #164198494]

danprocoder committed:
chore(createArticle): Written test

- Changed folder structure. Moved files from ./server/routes/api to ./server/routes
- Removed base controller class and made response helper a stand alone helper in ./server/utils/response.js

[Finish #164198494]

danprocoder committed:
chore(createArticle): Written test

[Finish #164198494]

danprocoder committed:
chore(articlesTest): Removed comment at end of page.

[Delivers #164198494]

danprocoder committed:
feat(*): Fixed merge conflict

- Fixed merge conflict
- Added DATABASE to travis script

[Delivers #164198494]
